nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -125,6 +126,16 @@ def create_weight_quantizer_insertion_command(
                                                                        quantizer_config,
                                                                        statistics)
 
+    @staticmethod
+    def unify_statistics(statistics: List[MinMaxTensorStatistic]) -> MinMaxTensorStatistic:
+        max_values, min_values = [], []
+        for statistic in statistics:
+            max_values.append(statistic.max_values)
+            min_values.append(statistic.min_values)
+        max_values = torch.max(torch.tensor(max_values))
+        min_values = torch.min(torch.tensor(min_values))
+        return PTMinMaxTensorStatistic(min_values=min_values, max_values=max_values)
```

Comment:
I think we could get rid of framework specific `TensorStatistic` classes by using framework specific tensor processor

Reply:
Yeah, it would be very cool to have the common `TensorStatistics`.